docs/functions-dev.md

```diff
@@ -42,3 +42,66 @@ To do so set the `--inspect` Node.js option when starting the functions server:
 - On Mac/Linux run `NODE_OPTIONS=--inspect netlify functions:serve`
 
 Then attach any Node.js debugger to the CLI process to debug your functions.
+
+
+## VS Code Debugging
+
+Put the below configuration in your repo's `.vscode/launch.json`.
+
+Then in `VS Code`:
+1. Open `Run and Debug` Sidebar. Make sure "Caught Exceptions" is deactivated.
+2. In the dropdown, select `Fullstack Debugging`
+3. Run the debugger. A chrome window should open. Wait until content is loaded (may take a few seconds for server to be ready).
+
+```jsonc
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  // ---- FULLSTACK NETLIFY DEV DEBUGGING ----
+  // 1. Make sure "Caught Exceptions" is deactivated in your VS Code debugger
+  // 2. Run 'Fullstack Debugging' from the debug interface
+  // 3. A chrome browser window will open, however it may take a few seconds
+  // before the server is ready and content is displayed.
+  "compounds": [
+    {
+      "name": "Fullstack Debugging",
+      "configurations": ["netlify dev", "client: chrome"]
+    }
+  ],
+  "configurations": [
+    {
+      "name": "client: chrome",
+      "type": "chrome",
+      "request": "launch",
+      "url": "http://localhost:8888",
+      "webRoot": "${workspaceFolder}",
+      "sourceMapPathOverrides": {
+        "webpack:///*": "${workspaceRoot}/*"
+      }
+    },
+    {
+      "name": "netlify dev",
+      "type": "node",
+      "request": "launch",
+      "skipFiles": ["<node_internals>/**"],
+      "program": "${workspaceFolder}/node_modules/.bin/netlify",
```

Comment:
As mentioned before this only works after installing the CLI locally so we should instruct users to do so.